src/blockdata/script.rs

```diff
@@ -1106,7 +1114,7 @@ mod test {
     use core::str::FromStr;
 
     use super::*;
-    use super::build_scriptint;
+    use super::write_scriptint;
```

Comment:
We can just delete these lines, wildcard import right above :) Probably not worth requiring re-acks just to do this though.